src/libs/CurrencyUtils.js

```diff
@@ -73,54 +73,49 @@ function isCurrencySymbolLTR(currencyCode) {
+ * This is because the backend always stores amounts in "cents". The backend works in integer cents to avoid precision errors
+ * when doing math operations.
  *
- * @note we do not currently support any currencies with more than two decimal places. Sorry Tunisia :(
+ * @note we do not currently support any currencies with more than two decimal places. Decimal past the second place will be rounded. Sorry Tunisia :(
  *
- * @param {String} currency
  * @param {Number} amountAsFloat
  * @returns {Number}
  */
-function convertToSmallestUnit(currency, amountAsFloat) {
-    const currencyUnit = getCurrencyUnit(currency);
-    // We round off the number to resolve floating-point precision issues.
-    return Math.round(amountAsFloat * currencyUnit);
+function convertToBackendAmount(amountAsFloat) {
+    return Math.round(amountAsFloat * 100);
 }
 
 /**
- * Takes an amount as an integer and converts it to a floating point amount.
- * For example, give [25, USD], return 0.25
- * Given [2550, USD], return 25.50
- * Given [2500, JPY], return 2500
+ * Takes an amount in "cents" as an integer and converts it to a floating point amount used in the frontend.
  *
  * @note we do not support any currencies with more than two decimal places.
  *
- * @param {String} currency
  * @param {Number} amountAsInt
  * @returns {Number}
  */
-function convertToWholeUnit(currency, amountAsInt) {
-    const currencyUnit = getCurrencyUnit(currency);
-    return Math.trunc(amountAsInt) / currencyUnit;
+function convertToFrontendAmount(amountAsInt) {
+    return Math.trunc(amountAsInt) / 100.0;
 }
 
 /**
- * Given an amount in the smallest units of a currency, convert it to a string for display in the UI.
+ * Given an amount in the "cents", convert it to a string for display in the UI.
+ * The backend always handle things in "cents" (subunit equal to 1/100)
  *
- * @param {Number} amountInSmallestUnit – should be an integer. Anything after a decimal place will be dropped.
+ * @param {Number} amountInCents – should be an integer. Anything after a decimal place will be dropped.
  * @param {String} currency
  * @returns {String}
  */
-function convertToDisplayString(amountInSmallestUnit, currency = CONST.CURRENCY.USD) {
-    const currencyUnit = getCurrencyUnit(currency);
-    const convertedAmount = Math.trunc(amountInSmallestUnit) / currencyUnit;
+function convertToDisplayString(amountInCents, currency = CONST.CURRENCY.USD) {
+    const convertedAmount = convertToFrontendAmount(amountInCents);
     return NumberFormatUtils.format(BaseLocaleListener.getPreferredLocale(), convertedAmount, {
         style: 'currency',
         currency,
+
+        // We are forcindg the number of decimals because we overridden the default number of decimals in the backend for RSD
+        // See: https://github.com/Expensify/PHP-Libs/pull/834
+        minimumFractionDigits: currency === 'RSD' ? getCurrencyDecimals(currency) : undefined,
```

Comment:
The library used in the backend and the library used in the frontend (Intl) format the RSD currency with 0 decimals. A client needed decimals, so we updated the backend to display RSD with two decimals.
Now, we have to force the frontend to also do the same so we don't get bug reports because of inconsistent behaviour.

I ONLY enabled this for RSD because otherwise the formatting will be inconsistent between backend and frontend.